application/controllers/admin/ParticipantsAction.php

```diff
@@ -1313,6 +1313,13 @@ public function blacklistControl()
      */
     public function storeBlacklistValues()
     {
+        $this->requirePostRequest();
+
+        if (!Permission::model()->hasGlobalPermission('settings', 'update')) {
```

Comment:
Not related to https://github.com/LimeSurvey/LimeSurvey/blob/590c1ac76e2da298d01c34ac4e9fe7f9138e343d/application/models/Permission.php#L869 `participantpanel` permission ?

Reply:
OK : settings !